unitTests/ui/RegisteredNamedMethodsTest.ts

```diff
@@ -278,6 +278,11 @@ export function RegisteredNamedMethodsTest() {
         expect(setup.cmp.client instanceof NoopAnalyticsClient).toBeTruthy();
       });
 
+      it('should update the search interface analytics client after being disabled', () => {
+        RegisteredNamedMethod.disableAnalytics(setup.env.root);
+        expect(setup.cmp.client).toBe(setup.env.searchInterface.usageAnalytics);
```

Comment:
I think a useful check in addition to the one already here is to verify that the client is an instance of the `NoopClient`.

Similarly for the test on 294, to check that it is an instance of the live client.

Reply:
That is exactly the previous test, should I merge them both?